launcher/InstanceImportTask.cpp

```diff
@@ -385,61 +388,83 @@ void InstanceImportTask::processFlame()
     connect(m_modIdResolver.get(), &Flame::FileResolvingTask::succeeded, [&]()
     {
         auto results = m_modIdResolver->getResults();
-        m_filesNetJob = new NetJob(tr("Mod download"), APPLICATION->network());
-        for(auto result: results.files)
-        {
-            QString filename = result.fileName;
-            if(!result.required)
-            {
-                filename += ".disabled";
+        //first check for blocked mods
+
+        QString text;
+        for(const auto& result: results.files.values()) {
+            if (!result.resolved || result.url.isEmpty()) {
+                text += QString("%1: <a href='%2'>%2</a><br/>").arg(result.fileName, result.websiteUrl);
             }
+        }
+        qWarning() << "Blocked mods found, displaying mod list";
+
+        auto message_dialog = new ScrollMessageBox(nullptr,
```

Comment:
```suggestion
        auto message_dialog = new ScrollMessageBox(m_parent,
```

A dialog having no parent can cause some weird behaviours, like showing up as a non-floating window in some tiling WMs, or displaying the dialog in weird positions

Reply:
I wondered if i was supposed to use it or not, i tried making the dialog into a modal one but it just didn't work. since it's marked as for removal i decided not to add one
Plus it works fine on my tiling WM

Reply:
Oh wow using show actually works for modal windows guess i was wrong, commiting a fix right now

Reply:
Ok you're right, the parent is not set where it should be (I added this when doing the Modrinth pack stuff, so I only added it to Modrinth stuff xd)

You just need to add `, this` to the `InstanceImportTask` construcotr call there and it should work:
https://github.com/timoreo22/PolyMC/blob/e62dcc3e8f9d2b63db80630963bb77db5b8b1ed9/launcher/ui/pages/modplatform/flame/FlamePage.cpp#L204

For reference, this is what it looks like without parent on my WM:
![image](https://user-images.githubusercontent.com/9145768/170682841-1d0e2ae9-778b-403f-abd7-0dd63b49458e.png)